app/src/main/scala/org/alephium/explorer/persistence/queries/ContractQueries.scala

```diff
@@ -135,4 +137,41 @@ object ContractQueries {
       .paginate(pagination)
       .asASE[Address](addressGetResult)
   }
+
+  def insertNFTCollectionMetadata(
+      contract: Address,
+      metadata: NFTCollectionMetadata
+  ): DBActionW[Int] = {
+    sqlu"""
+      INSERT INTO nft_collection_metadata (
+        "contract",
+        "collection_uri"
+        )
+      VALUES (${contract},${metadata.collectionUri})
+      ON CONFLICT
+      ON CONSTRAINT nft_collection_metadata_pkey
+      DO NOTHING
```

Comment:
Could [this](https://github.com/alephium/explorer-backend/pull/486#discussion_r1301861964) be replicate here?

Reply:
yes, I missed it